fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/flight_model.cfg

```diff
@@ -356,28 +356,28 @@ Valve.52 = Name:LeftOuterEmerTransferValve#OpeningTime:3#Circuit:52
 Valve.53 = Name:RightOuterEmerTransferValve#OpeningTime:3#Circuit:53
 Valve.54 = Name:GalleryAuxRefuelValveLeft#OpeningTime:3#Circuit:54
 Valve.55 = Name:GalleryAuxRefuelValveRight#OpeningTime:3#Circuit:55
-Valve.56 = Name:TransferDefuelValve#OpeningTime:3#Circuit:46
+Valve.56 = Name:TransferDefuelValve#OpeningTime:3#Circuit:56
 Valve.57 = Name:JettisonNozzleValveLeft#Circuit:57
 Valve.58 = Name:JettisonNozzleValveRight#Circuit:58
 
 Pump.1 = Name:Feed1TankPump1#Pressure:30#DestinationLine:Pump1Feed1ToJunc1#TankFuelRequired:Feed1#Type:Electric#Index:1
 Pump.2 = Name:Feed1TankPump2#Pressure:30#DestinationLine:Pump2Feed1ToJunc1#TankFuelRequired:Feed1#Type:Electric#Index:2
 Pump.3 = Name:Feed2TankPump1#Pressure:30#DestinationLine:Pump1Feed2ToJunc2#TankFuelRequired:Feed2#Type:Electric#Index:3
 Pump.4 = Name:Feed2TankPump2#Pressure:30#DestinationLine:Pump2Feed2ToJunc2#TankFuelRequired:Feed2#Type:Electric#Index:4
-Pump.5 = Name:Feed3TankPump1#Pressure:30#DestinationLine:Pump1Feed3ToJunc3#TankFuelRequired:Feed3#Type:Electric#Index:12
-Pump.6 = Name:Feed3TankPump2#Pressure:30#DestinationLine:Pump2Feed3ToJunc3#TankFuelRequired:Feed3#Type:Electric#Index:13
-Pump.7 = Name:Feed4TankPump1#Pressure:30#DestinationLine:Pump1Feed4ToJunc4#TankFuelRequired:Feed4#Type:Electric#Index:17
-Pump.8 = Name:Feed4TankPump2#Pressure:30#DestinationLine:Pump2Feed4ToJunc4#TankFuelRequired:Feed4#Type:Electric#Index:18
+Pump.5 = Name:Feed3TankPump1#Pressure:30#DestinationLine:Pump1Feed3ToJunc3#TankFuelRequired:Feed3#Type:Electric#Index:5
+Pump.6 = Name:Feed3TankPump2#Pressure:30#DestinationLine:Pump2Feed3ToJunc3#TankFuelRequired:Feed3#Type:Electric#Index:6
+Pump.7 = Name:Feed4TankPump1#Pressure:30#DestinationLine:Pump1Feed4ToJunc4#TankFuelRequired:Feed4#Type:Electric#Index:7
+Pump.8 = Name:Feed4TankPump2#Pressure:30#DestinationLine:Pump2Feed4ToJunc4#TankFuelRequired:Feed4#Type:Electric#Index:8
 Pump.9 = Name:LeftOuterTankPump#Pressure:20#DestinationLine:LeftOuterTankPumpToFwdGalleryJunction#TankFuelRequired:LeftOuter#Type:Electric#Index:9
 Pump.10 = Name:LeftMidTankPumpFwd#Pressure:20#DestinationLine:LeftMidTankPumpFwdToFwdGalleryJunction#TankFuelRequired:LeftMid#Type:Electric#Index:10
 Pump.11 = Name:LeftMidTankPumpAft#Pressure:20#DestinationLine:LeftMidTankPumpAftToAftGalleryJunction#TankFuelRequired:LeftMid#Type:Electric#Index:11
-Pump.12 = Name:LeftInnerTankPumpFwd#Pressure:20#DestinationLine:LeftInnerTankPumpFwdToFwdGalleryJunction#TankFuelRequired:LeftInner#Type:Electric#Index:5
-Pump.13 = Name:RightInnerTankPumpFwd#Pressure:20#DestinationLine:RightInnerTankPumpFwdToFwdGalleryJunction#TankFuelRequired:RightInner#Type:Electric#Index:7
+Pump.12 = Name:LeftInnerTankPumpFwd#Pressure:20#DestinationLine:LeftInnerTankPumpFwdToFwdGalleryJunction#TankFuelRequired:LeftInner#Type:Electric#Index:12
+Pump.13 = Name:RightInnerTankPumpFwd#Pressure:20#DestinationLine:RightInnerTankPumpFwdToFwdGalleryJunction#TankFuelRequired:RightInner#Type:Electric#Index:17
```

Comment:
I wanna say this is correct because fuel pump circuit 17 corresponds to the right inner pump to the forward gallery

https://github.com/flybywiresim/aircraft/blob/c4ab9895fb290e60a6c0bc80e809421b47f76bb7/fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/systems.cfg#L452

Reply:
It's a bit confusing that these are the only 2 pumps which have different electrical indices but it's ok.